fix(sdk):  handle 'personal' for collectionId when creating dashboards and in the collection browser (#53553)
metabase/metabase: docs/embedding/sdk/dashboards.md

Props:

| Prop         | Type                             | Description                                                                                                 |
| ------------ | -------------------------------- | ----------------------------------------------------------------------------------------------------------- |
| name         | `string`                         | (required) Dashboard title                                                                                  |
| description  | `string \| null`                 | Optional dashboard description                                                                              |

> | collectionId | `number \| 'root' \| 'personal'` | Collection where to create a new dashboard. You can use predefined system values like `root` or `personal`. |

Reviewer: ```suggestion
| collectionId | `number \| 'root' \| 'personal'` | Collection in which to create a new dashboard. You can use predefined system values like `root` or `personal`. |
```

Supported component props:

| Prop          | Type                             | Description                                                                                     |
| ------------- | -------------------------------- | ----------------------------------------------------------------------------------------------- |

> | collectionId? | `number \| 'root' \| 'personal'` | Initial collection field value. You can use predefined system values like `root` or `personal`. |

Reviewer: ```suggestion
| collectionId? | `number \| 'root' \| 'personal'` | Initial collection in which to create a dashboard. You can use predefined system values like `root` or `personal`. |
```
